src/vm/eventpipe.h

```diff
@@ -29,6 +29,70 @@ typedef void (*EventPipeCallback)(
     void *FilterData,
     void *CallbackContext);
 
+struct EventData
+{
+public:
+    unsigned long ptr;
```

Comment:
Addressed in https://github.com/dotnet/coreclr/pull/13347/commits/851943c5d5dc653dc1625a376c26bc2d6cf6a587